modules/react/tabs/lib/TabsItem.tsx

```diff
@@ -81,11 +81,11 @@ export interface TabsItemProps
   tabIndex?: number;
 }
 
-export const StyledTabItem = styled(Box.as('button'))<StyledType & Pick<TabsItemProps, 'spacing'>>(
-  ({theme, spacing}) => ({
+export const StyledTabItem = styled(Box.as('button'))<StyledType & Pick<TabsItemProps, 'gap'>>(
```

Comment:
This won't work, but it's a weird case. I'm not immediately sure what the solution is. It might need to become a `Grid` to support the `gap` property and handle styling correctly.

Reply:
I'll wait for this then once we have our discussion later for the codemod